torch/csrc/jit/passes/python_print.cpp

```diff
@@ -720,6 +719,9 @@ struct PythonPrintPass {
         stmt << "annotate(" << node->output()->type()->python_str() << ", "
              << useOf(node->input()) << ")";
       } break;
+      case prim::_unchecked_unwrap_optional: {
```

Comment:
(maybe in a followup?) We could have the Python Printer spit out `prim::__is__` nodes as Python `is` statements and then skip emitting these altogether so they'd get re-inserted when the code is read in again